base_view_inheritance_extension/models/ir_ui_view.py

```diff
@@ -41,12 +41,24 @@ def __missing__(self, key):
 class IrUiView(models.Model):
     _inherit = 'ir.ui.view'
 
+    user_ids = fields.Many2many('res.users', string='Restrict to users')
+
     @api.model
     def apply_inheritance_specs(self, source, specs_tree, inherit_id):
         for specs, handled_by in self._iter_inheritance_specs(specs_tree):
```

Comment:
question/ explanation:   _iter_inheritance_specs  returns the node and the handler, 
 _iter_inheritance_specs then iterates and yields through the tree and computes node/attributes and finally returns, but i see no  new attribute user_id= to be handled. So is this intended not to be used in XML as a new attribute but just to impose via code some user_ids for certain XMLids, that will then be popped out get_inheriting_views_arch?

Or have i misunderstood?